external-crates/move/crates/move-analyzer/src/symbols.rs

```diff
@@ -2478,6 +2594,7 @@ fn assert_use_def_with_doc_string(
     use_idx: usize,
     use_line: u32,
     use_col: u32,
+    use_file: &str,
```

Comment:
These changes (adding another parameter and more verbose assertions) is not related to enhancing on-hover for consts, but make debugging/adding the tests much easier.